update FAQ visibility (#110)
glpi-project/doc: source/modules/tools/knowledgebase.rst

.. note::
   Only public FAQ items are visible to users of simplified interface. Other elements are visible only to technicians via standard interface.

> Each article of the knowledge base must have one or several targets, a target being either an entity, a group, a profile or a user, to be readable. As long as an article has no target, it is visible only by its writer, is flagged as `unpublished` and appears in table `Unpublished articles` on the home page of the knowledge base.

Reviewer: ```suggestion
Each article of the knowledge base must have one or more targets, being entities, groups, profiles or users, to be readable. As long as an article has no target, it is visible only by its writer, is flagged as `unpublished` and appears in table `Unpublished articles` on the home page of the knowledge base.
```

.. note::
   .. versionchanged:: 10.1

   Targeting applies in the same way whether the article is in the FAQ or only in the knowledge base.

.. note::

>    You can publish an article to everyone, targeting the root entity.

Reviewer: ```suggestion
   You can publish an article for everyone by targeting the root entity.
```
